rdd/collect/CollectExample.py

```diff
@@ -0,0 +1,9 @@
+from pyspark import SparkContext
+
+if __name__ == "__main__":
+    sc = SparkContext("local", "collect")
```

Comment:
Please set the logging level to ERROR similar to what the Scala problem does to reduce the noise of the output

Reply:
Hi James, some considerations about the logging level when using pyspark:
 - From the script itself, when using pyspark, we can only set the log level after starting the SparkContext, this means that logs printed when the SparkContext is starting will be printed anyway.
 - The best way to reduce the noise of the output is to configure the file log4j.properties inside spark/conf folder.
That being said, I will set the log levels to ERROR after the SparkContext starts